apps/api/src/database/migrations/1730724383210-ArchiveCompletedNotifications.ts

```diff
@@ -0,0 +1,166 @@
+import { MigrationInterface, QueryRunner, Table, TableForeignKey } from 'typeorm';
+
+export class ArchiveCompletedNotifications1730724383210 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Drop foreign key from notify_notification_retries
+    const table = await queryRunner.getTable('notify_notification_retries');
+    const foreignKey = table.foreignKeys.find(
+      (fk) => fk.columnNames.indexOf('notification_id') !== -1,
+    );
+    await queryRunner.dropForeignKey('notify_notification_retries', foreignKey);
+
+    // Create table archived_notifications
+    await queryRunner.createTable(
+      new Table({
+        name: 'archived_notifications',
```

Comment:
First comment left in the task suggested this name `archived_notifications` only so used it, if needed can change it

Reply:
Yeah that is table name but we should always use prefix to avoid any conflicts with host database tables. THis prefix was added to differentiate our osmox tables from host database tables if we decided to use host database.

Reply:
fixed 384188b31ee2fe3ccd8a2fa70b31f43faab525c1